keymap.cson

```diff
@@ -23,3 +23,14 @@
 # This file uses CoffeeScript Object Notation (CSON).
 # If you are unfamiliar with CSON, you can read more about it here:
 # https://github.com/bevry/cson#what-is-cson
+
+# window/pane navigation
+'body, atom-workspace atom-text-editor:not([mini])':
+  'cmd-h': 'window:focus-previous-pane'
+  'cmd-l': 'window:focus-next-pane'
+  'cmd-shift-v': 'pane:split-right'
+  'cmd-shift-s': 'pane:split-down'
+  'cmd-shift-h': 'window:focus-pane-on-left'
+  'cmd-shift-l': 'window:focus-pane-on-right'
```

Comment:
and this takes over "split selection into lines"... and cmd-L is select line, which is pretty common.

I think we may need to go to leaders for these. I'm good w/ what I proposed (usng cmd-k). Not sure what to do for the next/prev pane though. seems most things cmd-_ are taken